packages/tailwindcss/src/candidate.ts

```diff
@@ -552,6 +564,9 @@ export function parseVariant(variant: string, designSystem: DesignSystem): Varia
 
     let selector = decodeArbitraryValue(variant.slice(1, -1))
```

Comment:
kinda bothers me that this var is called `selector` and not `arbitraryValue` like in the 5 other places that look identical haha

Reply:
I mean, in this case it _is_ a selector because of the variant :P 

```
[.foo]:…
```